### PR TITLE
Rework docs on dynamic logging level threshold

### PR DESCRIPTION
**Pull request by christiand93**
- Add command and description to generate key-pair with correct format.
- Provide commands and description required to use token-creator tool
- Minor changes and format fixes.

### CODE REVIEW

File: docs/advanced-usage/04-dynamic-logging-level-threshold.md [hunk 1]
```diff
@@ -23,54 +23,94 @@ It has no effect on the level reported as part of the logs.
...
-You can create JWTs by using the [TokenCreator](https://github.com/SAP/cf-nodejs-logging-support/tree/master/tools/token-creator) from the tools folder.
+You can change the logging level threshold for a specific request by providing a JSON Web Token ([JWT](https://de.wikipedia.org/wiki/JSON_Web_Token)) via the request header.
+Using this feature allows changing the logging level threshold dynamically without the need to redeploy your app.
 
-Basically, JWTs are signed using RSA or HMAC signing algorithms. 
-But we decided to support RSA algorithms (RS256, RS384 and RS512) only. 
+### 1 Creating a key-pair
+
+In order to sign and verify JWTs a PEM encoded private key and a matching public key is required.
+You can create a key-pair using following command:
```
Comment (juergen-walter) [inline docs/advanced-usage/04-dynamic-logging-level-threshold.md:33]: Improve wording
```suggestion
To sign and verify JWTs a PEM encoded private key and a matching public key is required.
You can create a key-pair using the following command:
```

File: docs/advanced-usage/04-dynamic-logging-level-threshold.md [hunk 2]
```diff
@@ -23,54 +23,94 @@ It has no effect on the level reported as part of the logs.
...
+
+```sh
+openssl rsa -in private.pem -outform PEM -pubout -out public.pem
+```
+
+The generated key-pair can be found in `private.pem` and `public.pem` files.
+
+### 2 Creating a JWT
+
+JWTs are signed claims, which consist of a header, a payload and a signature.
```
Comment (juergen-walter) [inline docs/advanced-usage/04-dynamic-logging-level-threshold.md:43]: Oxford comma ;-) 
```suggestion
JWTs are signed claims, which consist of a header, a payload, and a signature.
```

File: docs/advanced-usage/04-dynamic-logging-level-threshold.md [hunk 3]
```diff
@@ -23,54 +23,94 @@ It has no effect on the level reported as part of the logs.
...
+node token-creator.js -f <path private.pem> -v <validity period> -i <issuer> <level>
+```
+
+The `<validity period>` sets the number of days the JWT will be valid.
+Once the created JWT expired, it can no longer be used for setting logging level threshold.
+Provide a numeric input for this placeholder.
+
+Provide a valid e-mail address for the `<issuer>` parameter.
+
+Specify one of the seven supported logging levels for the `<level>` argument: *off*, *error*, *warn*, *info*, *verbose*, *debug* and *silly*.
```
Comment (juergen-walter) [inline docs/advanced-usage/04-dynamic-logging-level-threshold.md:62]: Oxford comma
'```suggestion
Specify one of the seven supported logging levels for the `<level>` argument: *off*, *error*, *warn*, *info*, *verbose*, *debug*, and *silly*.
```

File: docs/advanced-usage/04-dynamic-logging-level-threshold.md [hunk 4]
```diff
@@ -23,54 +23,94 @@ It has no effect on the level reported as part of the logs.
...
+
+The payload of the created JWT has following structure:
```
Comment (juergen-walter) [inline docs/advanced-usage/04-dynamic-logging-level-threshold.md:64]:
```suggestion
The payload of the created JWT has the following structure:
```

File: docs/advanced-usage/04-dynamic-logging-level-threshold.md [hunk 5]
```diff
@@ -23,54 +23,94 @@ It has no effect on the level reported as part of the logs.
...
+Redeploy your app after setting the environment variable.
+
+### 4 Attaching JWTs to requests
 
-### 3 Attaching JWTs to requests
-Provide the created JWTs via a header field named 'SAP-LOG-LEVEL'. The logging level threshold will be set to the provided level for this request and corresponding custom log messages. 
+Provide the created JWT via a header field named 'SAP-LOG-LEVEL'. The logging level threshold will be set to the provided level for this request and corresponding custom log messages.
 
-Note: If the provided JWT cannot be verified, is expired or contains an invalid logging level, the library ignores it and uses the global logging level threshold.
+**Note**: If the provided JWT cannot be verified, is expired or contains an invalid logging level, the library ignores it and uses the global logging level threshold.
```
Comment (juergen-walter) [inline docs/advanced-usage/04-dynamic-logging-level-threshold.md:100]: Comma
```suggestion
**Note**: If the provided JWT cannot be verified, is expired, or contains an invalid logging level, the library ignores it and uses the global logging level threshold.
```

**Review by juergen-walter — APPROVED** [review]
